x86_symbolic/tests/Main.hs

```diff
@@ -103,23 +104,77 @@ main = do
+          , TTH.testCase "rsp" $ getRegName MXSR.rsp TTH.@?= "r!rsp"
+          , TTH.testCase "rbp" $ getRegName MXSR.rbp TTH.@?= "r!rbp"
+          , TTH.testCase "rsi" $ getRegName MXSR.rsi TTH.@?= "r!rsi"
+          , TTH.testCase "rdi" $ getRegName MXSR.rdi TTH.@?= "r!rdi"
+          , TTH.testCase "r8" $ getRegName MXSR.r8 TTH.@?= "r!r8"
+          , TTH.testCase "r9" $ getRegName MXSR.r9 TTH.@?= "r!r9"
+          , TTH.testCase "r10" $ getRegName MXSR.r10 TTH.@?= "r!r10"
+          , TTH.testCase "r11" $ getRegName MXSR.r11 TTH.@?= "r!r11"
+          , TTH.testCase "r12" $ getRegName MXSR.r12 TTH.@?= "r!r12"
+          , TTH.testCase "r13" $ getRegName MXSR.r13 TTH.@?= "r!r13"
+          , TTH.testCase "r14" $ getRegName MXSR.r14 TTH.@?= "r!r14"
+          , TTH.testCase "r15" $ getRegName MXSR.r15 TTH.@?= "r!r15"
+          , TTH.testCase "cf" $ getRegName MXSR.cf TTH.@?= "r!cf"
+          , TTH.testCase "pf" $ getRegName MXSR.pf TTH.@?= "r!pf"
+          , TTH.testCase "af" $ getRegName MXSR.af TTH.@?= "r!af"
+          , TTH.testCase "zf" $ getRegName MXSR.zf TTH.@?= "r!zf"
+          , TTH.testCase "sf" $ getRegName MXSR.sf TTH.@?= "r!sf"
+          , TTH.testCase "tf" $ getRegName MXSR.tf TTH.@?= "r!tf"
+          , TTH.testCase "if" $ getRegName MXSR.if_ TTH.@?= "r!if"
+          , TTH.testCase "df" $ getRegName MXSR.df TTH.@?= "r!df"
+          , TTH.testCase "of" $ getRegName MXSR.of_ TTH.@?= "r!of"
+          , TTH.testCase "ie" $ getRegName MXSR.ie TTH.@?= "r!ie"
+          , TTH.testCase "de" $ getRegName MXSR.de TTH.@?= "r!de"
+          , TTH.testCase "ze" $ getRegName MXSR.ze TTH.@?= "r!ze"
+          , TTH.testCase "oe" $ getRegName MXSR.oe TTH.@?= "r!oe"
+          , TTH.testCase "ue" $ getRegName MXSR.ue TTH.@?= "r!ue"
+          , TTH.testCase "pe" $ getRegName MXSR.pe TTH.@?= "r!pe"
+          , TTH.testCase "ef" $ getRegName MXSR.ef TTH.@?= "r!ef"
+          , TTH.testCase "es" $ getRegName MXSR.es TTH.@?= "r!es"
+          , TTH.testCase "c0" $ getRegName MXSR.c0 TTH.@?= "r!c0"
+          , TTH.testCase "c1" $ getRegName MXSR.c1 TTH.@?= "r!c1"
+          , TTH.testCase "c2" $ getRegName MXSR.c2 TTH.@?= "r!c2"
+          , TTH.testCase "c3" $ getRegName MXSR.c3 TTH.@?= "r!c3"
+          , TTH.testCase "top" $ getRegName MXSR.top TTH.@?= "r!x87Top"
+          , TTH.testCase "tag0" $ getRegName MXSR.tag0 TTH.@?= "r!x87Tag0"
+          , TTH.testCase "tag1" $ getRegName MXSR.tag1 TTH.@?= "r!x87Tag1"
+          , TTH.testCase "tag2" $ getRegName MXSR.tag2 TTH.@?= "r!x87Tag2"
+          , TTH.testCase "tag3" $ getRegName MXSR.tag3 TTH.@?= "r!x87Tag3"
+          , TTH.testCase "tag4" $ getRegName MXSR.tag4 TTH.@?= "r!x87Tag4"
+          , TTH.testCase "tag5" $ getRegName MXSR.tag5 TTH.@?= "r!x87Tag5"
+          , TTH.testCase "tag6" $ getRegName MXSR.tag6 TTH.@?= "r!x87Tag6"
+          , TTH.testCase "tag7" $ getRegName MXSR.tag7 TTH.@?= "r!x87Tag7"
+          , TTH.testCase "st0" $ getRegName MXSR.st0 TTH.@?= "r!stmm0"
```

Comment:
I've seen this register referred to as both `st0` and `mm0`, but never as `stmm0`. I think we should pick one convention or the other.